zhmcclient/_session.py

```diff
@@ -75,6 +75,11 @@ def __init__(self, host, userid=None, password=None):
           password (:term:`string`):
             Password of the HMC user to be used.
 
+          session_id (:term:`string`):
+            An opaque string that provides a cryptographically
+            strong identifier of the API session (known as a session id)
+            under which this request is executed.
+
```

Comment:
Some comments on the description of the new `session_id` argument of `Session.__init__()`:

1. On "An opaque string that provides a cryptographically strong identifier of the API session (known as a session id)": I suggest to use the same wording we use for the `session_id` property: "session ID".

2. The description does not make it clear enough that providing a session ID puts the session into the logged-on state.

3. It is not clear how the `session_id` and the `userid` / `password` arguments play together.
 
I suggest the following description for the `session_id` argument, to address these issues:

```
    A session ID returned by the HMC that is to be used for this session.
    Supplying a session ID puts the session into a logged-on state.

    The `userid` and `password` arguments may be omitted if a session ID
    is supplied. If they are omitted, expiration of the session ID causes an exception
    to be raised by subsequent operations, instead of performing an automatic
    re-logon.
```